<ui version="4.0" >
 <class>PageQuick</class>
 <widget class="QWidget" name="PageQuick" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Quick Build Command</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton1" >
          <property name="text" >
           <string>LaTeX + dvips + View PS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton2" >
          <property name="text" >
           <string>LaTeX + View DVI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton3" >
          <property name="text" >
           <string>PdfLaTeX + View PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton4" >
          <property name="text" >
           <string>LaTeX + dvipdfm + View PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton5" >
          <property name="text" >
           <string>LaTeX + dvips + ps2pdf + View PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton6" >
          <property name="text" >
           <string>User : (% : filename without extension)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditUserquick" />
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>(the commands must be separated by '|')</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>401</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
